components/layout/display_list_builder.rs

```diff
@@ -1506,6 +1552,10 @@ impl BlockFlowDisplayListBuilding for BlockFlow {
     }
 
     fn will_get_layer(&self) -> bool {
+        // TODO(gw): The first condition below was previously valid as
+        // this was only called from absolute block function. But now
+        // it is called from static block too - so probably needs an
+        // additional check.
```

Comment:
Hmm, I think some tests would fail if this were incorrect. I think `NEEDS_LAYER` will not be set for static blocks unless they actually need a layer, right?